test/components/views/settings/tabs/user/__snapshots__/KeyboardUserSettingsTab-test.tsx.snap

```diff
@@ -140,7 +140,7 @@ exports[`KeyboardUserSettingsTab renders list of keyboard shortcuts 1`] = `
             <div
               className="mx_KeyboardShortcut_shortcutRow"
             >
-              Cancel replying to a message
+              missing translation: en|Cancel replying to a message
```

Comment:
Those should not be expected in the snapshot test output.
We should aim to have the same output as before your changes

Reply:
it seems to me that the snapshot test does not handle translation, so the output is inevitably modified for those pieces of text that are now translated. Besides we already have this kind of output for the shortcuts themselves, which were already correctly translated (ex: https://github.com/matrix-org/matrix-react-sdk/blob/38a57c8b4f28264674de0c993b2ced4c2950c704/test/components/views/settings/tabs/user/__snapshots__/KeyboardUserSettingsTab-test.tsx.snap#L154)

...or I'm missing something

Reply:
Oh interesting, I thought that would have been handled! My bad, let me change my review.
That would make sense that with your changes those strings are now translated strings hence the new `missing translation: en|`